python/dolfinx/io/gmshio.py

```diff
@@ -114,10 +130,13 @@ def extract_topology_and_markers(model, name: typing.Optional[str] = None):
             model will be used.
 
     Returns:
-        A nested dictionary where each key corresponds to a gmsh cell
+        A tuple ``(topologies, groups)``, where ``topologies`` is a
+        nested dictionary where each key corresponds to a gmsh cell
         type. Each cell type found in the mesh has a 2D array containing
         the topology of the marked cell and a list with the
-        corresponding markers.
+        corresponding markers. ``groups`` is a dictionary where the key
```

Comment:
Name `groups` to `physical_groups` here so the naming is consistent throughout.

Reply:
Fixed in https://github.com/FEniCS/dolfinx/pull/3547/commits/205662e5ab99923a7727cbf558dee91c99cc204c